src/asyncWithLDProvider.tsx

```diff
@@ -41,11 +41,12 @@ export default async function asyncWithLDProvider(config: AsyncProviderConfig) {
   );
 
   const initialFlags = options?.bootstrap && options.bootstrap !== 'localStorage' ? options.bootstrap : fetchedFlags;
+  let currentFlags = initialFlags;
```

Comment:
@yusinto 

Thank you for your suggestion! After I changed to use `initialFlags`, I felt the variable was no longer `initial`. So I tried to use `currentFlags`. Please could you have a look at it? If it's better to use `initialFlags` instead of `currentFlags`, please let me know and I would change it.